ml_ops.sh

```diff
@@ -21,6 +21,16 @@ export LUSER
 export TOL
 export KRB_AUTH
 
+#check for missing variables
+if [[ -z $FDATE || -z $YR || -z $MH || -z $DY || -z $DSOURCE || -z $TOL ]];
```

Comment:
is it possible to construct the FDATE value from YR, MH, and DY, so we can save the redundancy in the call?

also, TOL is not explained